packages/cli/src/index.ts

```diff
@@ -341,7 +341,20 @@ const main = async () => {
         // SSO login, so set the current scope to the appropriate Team
         client.config.currentTeam = result.teamId;
       } else {
-        delete client.config.currentTeam;
+        let user = null;
+        try {
+          user = await getUser(client);
+        } catch (err: unknown) {
+          // Shouldn't happen since we just logged in
+          console.error(error('Not able to load user'));
```

Comment:
Same as https://github.com/vercel/vercel/pull/10535#discussion_r1330333984

```suggestion
          console.error(error('Failed to fetch the logged in user. Please try again.'));
```